backend/constant/errs.go

```diff
@@ -37,22 +37,23 @@ var (
 
 // api
 var (
-	ErrTypeInternalServer     = "ErrInternalServer"
-	ErrTypeInvalidParams      = "ErrInvalidParams"
-	ErrTypeNotLogin           = "ErrNotLogin"
-	ErrTypePasswordExpired    = "ErrPasswordExpired"
-	ErrNameIsExist            = "ErrNameIsExist"
-	ErrDemoEnvironment        = "ErrDemoEnvironment"
-	ErrCmdIllegal             = "ErrCmdIllegal"
-	ErrXpackNotFound          = "ErrXpackNotFound"
-	ErrXpackNotActive         = "ErrXpackNotActive"
-	ErrXpackLost              = "ErrXpackLost"
-	ErrXpackTimeout           = "ErrXpackTimeout"
-	ErrXpackOutOfDate         = "ErrXpackOutOfDate"
-	ErrApiConfigStatusInvalid = "ErrApiConfigStatusInvalid"
-	ErrApiConfigKeyInvalid    = "ErrApiConfigKeyInvalid"
-	ErrApiConfigIPInvalid     = "ErrApiConfigIPInvalid"
-	ErrApiConfigDisable       = "ErrApiConfigDisable"
+	ErrTypeInternalServer      = "ErrInternalServer"
+	ErrTypeInvalidParams       = "ErrInvalidParams"
+	ErrTypeNotLogin            = "ErrNotLogin"
+	ErrTypePasswordExpired     = "ErrPasswordExpired"
+	ErrNameIsExist             = "ErrNameIsExist"
+	ErrDemoEnvironment         = "ErrDemoEnvironment"
+	ErrCmdIllegal              = "ErrCmdIllegal"
+	ErrXpackNotFound           = "ErrXpackNotFound"
+	ErrXpackNotActive          = "ErrXpackNotActive"
+	ErrXpackLost               = "ErrXpackLost"
+	ErrXpackTimeout            = "ErrXpackTimeout"
+	ErrXpackOutOfDate          = "ErrXpackOutOfDate"
+	ErrApiConfigStatusInvalid  = "ErrApiConfigStatusInvalid"
+	ErrApiConfigKeyInvalid     = "ErrApiConfigKeyInvalid"
+	ErrApiConfigIPInvalid      = "ErrApiConfigIPInvalid"
+	ErrApiConfigDisable        = "ErrApiConfigDisable"
+	ErrApiConfigKeyTimeInvalid = "ErrApiConfigKeyTimeInvalid"
 )
 
 // app
```

Comment:
The code snippet you provided does not contain any major irregularities, potential issues, or significant optimizations to suggest. There are just minor formatting updates made between the third and fourth line of code comments. The changes include adding an ellipsis (`...`) at the end of each comment to align with the previous comments' format. These changes do not affect the functionality of the code and are considered stylistic adjustments rather than functional improvements.

If there is anything else specific you would like checked regarding this or other code snippets, please let me know!